### PR TITLE
Add changed class and component for manage waiter

### PR DESCRIPTION
**Pull request by taniarezhytska**

### CODE REVIEW

File: TAF_EasyRest/TestFramework/PageComponents/CreateNewWaiterPageComponent.cs [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+{
+    public class CreateNewWaiterPageComponent : BasePage
+    {
+        public CreateNewWaiterPageComponent(IWebDriver driver) : base(driver) { }
+
+        private IWebElement _nameField => driver.FindElement(By.Name("name"));
+        private IWebElement _emailField => driver.FindElement(By.Name("email"));
+        private IWebElement _passwordField => driver.FindElement(By.Name("password"));
+        private IWebElement _phoneNumberField => driver.FindElement(By.Name("phone_number"));
+        private IWebElement _addToSubmitRegistrationWaiterButton => driver.FindElement(By.XPath("//span[contains(text(),'Add')]/parent::button"));
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/PageComponents/CreateNewWaiterPageComponent.cs:11]: Change name

File: TAF_EasyRest/TestFramework/PageComponents/CreateNewWaiterPageComponent.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        private IWebElement _cancelRegistrationWaiterButton => driver.FindElement(By.XPath("//span[contains(text(),'Cancel')]/parent::button"));
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/PageComponents/CreateNewWaiterPageComponent.cs:12]: Change name

File: TAF_EasyRest/TestFramework/PageComponents/CreateNewWaiterPageComponent.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        {
+            _passwordField.Click();
+            return this;
+        }
+        public CreateNewWaiterPageComponent ClickPhoneNumberField()
+        {
+            _phoneNumberField.Click();
+            return this;
+        }
+        public CreateNewWaiterPageComponent ClickAddToSubmitRegistrationWaiterButton()
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/PageComponents/CreateNewWaiterPageComponent.cs:36]: Name...

File: TAF_EasyRest/TestFramework/PageComponents/CreateNewWaiterPageComponent.cs [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        {
+            _addToSubmitRegistrationWaiterButton.Click();
+            return this;
+        }
+        public CreateNewWaiterPageComponent ClickCancelRegistrationWaiterButton()
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/PageComponents/CreateNewWaiterPageComponent.cs:41]: Name...

File: TAF_EasyRest/TestFramework/PageComponents/PersonalInfoPageComponent.cs [hunk 5]
```diff
@@ -1,34 +1,34 @@
 ﻿
 namespace TestFramework.PageComponents
 {
-    public class PersonalInfoPageComponent : BasePage
+    public class ManageWa : BasePage
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/PageComponents/PersonalInfoPageComponent.cs:4]: Name...

File: TAF_EasyRest/TestFramework/Pages/CurrentOrdersPage.cs [hunk 6]
```diff
@@ -6,11 +6,11 @@ namespace TestFramework.Pages
     public class CurrentOrdersPage : BasePage
     {
         public NavigationMenuPageComponent NavigationMenuPageComponent { get; }
-        public PersonalInfoPageComponent PersonalInfoPageComponent { get; }
+        public ManageWa PersonalInfoPageComponent { get; }
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/Pages/CurrentOrdersPage.cs:9]: Name...

File: TAF_EasyRest/TestFramework/Pages/ManageWaitersPage.cs [hunk 7]
```diff
@@ -1,86 +1,23 @@
...
     {
         public ManageWaitersPage(IWebDriver driver) : base(driver)
         {
-            personalInfoPageComponent = new PersonalInfoPageComponent(driver);
+            personalInfoPageComponent = new ManageWa(driver);
         }
-        public PersonalInfoPageComponent personalInfoPageComponent { get; }
+        public ManageWa personalInfoPageComponent { get; }
 
         private IWebElement _removeWaiterButton(string username) => driver.FindElement(By.XPath($"//span[contains(text(), '{username}')]/parent::div//following-sibling::button"));
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/Pages/ManageWaitersPage.cs:12]: Delete this button.

File: TAF_EasyRest/TestFramework/Pages/OwnerPanelRestaurantsPage.cs [hunk 8]
```diff
@@ -6,11 +6,11 @@ namespace TestFramework.Pages
     public class OwnerPanelRestaurantsPage : BasePage
     {
         public NavigationMenuPageComponent NavigationMenuPageComponent { get; }
-        public PersonalInfoPageComponent PersonalInfoPageComponent { get; }
+        public ManageWa PersonalInfoPageComponent { get; }
```
Comment (LoveMyDaughter) [inline TAF_EasyRest/TestFramework/Pages/OwnerPanelRestaurantsPage.cs:9]: Name...

**Review by LoveMyDaughter — CHANGES_REQUESTED** [review]
Please make changes.

File: TAF_EasyRest/TestFramework/PageComponents/CreateNewWaiterPageComponent.cs [hunk 9]
```diff
@@ -0,0 +1,74 @@
+﻿namespace TestFramework.PageComponents
+{
+    public class CreateNewWaiterPageComponent 
+    {
+        IWebDriver driver { get; }
```
Comment (dmatsev) [inline TAF_EasyRest/TestFramework/PageComponents/CreateNewWaiterPageComponent.cs:5]: please specify the access modifier

**Pull request closed without merging**